src/createServer.js

```diff
@@ -1,3 +1,59 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+function createServer() {
+  return http.createServer((request, response) => {
+    response.setHeader('Content-Type', 'application/json');
+
+    const url = new URL(request.url, `http://${request.headers.host}`);
+    const text = url.pathname.slice(1);
+    const toCase = url.searchParams.get('toCase');
+
+    const validCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
```

Comment:
Better way use VALID_CASE for constant name.
Also, move all constants (status code, error messages, cases) to upper scope or better way - to separate file.